plugins/destination/postgresql/main.go

```diff
@@ -2,13 +2,19 @@ package main
 
 import (
 	"github.com/cloudquery/cloudquery/plugins/destination/postgresql/client"
+	"github.com/cloudquery/plugin-sdk/plugins"
 	"github.com/cloudquery/plugin-sdk/serve"
 )
 
 const (
 	sentryDSN = "https://19d1257d36854a51b17c06614e76dc2d@o1396617.ingest.sentry.io/4503896817336320"
 )
 
+var (
+	Version = "Development"
```

Comment:
I see. ok I'll move it that folder for now even though for destination it's a bit weird but not a big deal.

Reply:
maybe we can actually do one for destinations ? as we don't really have resources there so that's a bit weird? maybe later we can come up with a convention that works for both.

Reply:
Yeah, I think it's not a blocker for now (the version won't get embedded, but it won't fail the release)